interfaces/builtin/screen_inhibit_control.go

```diff
@@ -55,10 +55,19 @@ dbus (send)
     bus=session
     path=/Screensaver
     interface=org.freedesktop.ScreenSaver
-    member=org.freedesktop.ScreenSaver.{Inhibit,UnInhibit}
+    member=org.freedesktop.ScreenSaver.{Inhibit,UnInhibit,SimulateUserActivity}
+    peer=(label=unconfined),
+
+# gnome, kde and cinnamon screensaver
+dbus (send)
+    bus=session
+    path=/{ScreenSaver,}
```

Comment:
Can yoy change this to be: `path=/{,ScreenSaver}`. The rule is equivalent but follows the rules style we've established.